docs/README.md

```diff
@@ -6,6 +6,11 @@ Formatted doc is [here](http://appium.io/documentation.html?lang=en).
 
 ## doc translation
 
-At the moment we are focusing on improving the english documentation. When it 
-is done we are planning to setup proper i18n l10n processes so that it is 
+At the moment we are focusing on improving the english documentation. When it
+is done we are planning to setup proper i18n l10n processes so that it is
```

Comment:
```suggestion
At the moment we are focusing on improving the English documentation. When it
is done, we are planning to establish proper i18n and l10n processes so that it is
```